docs/specs/legacy.yml

```diff
@@ -62,7 +62,6 @@ outputs:
         <meta name=\"search.ms_product\" content=\"\" />
         <meta name=\"search.ms_sitename\" content=\"Docs\" />
         <meta name=\"site_name\" content=\"Docs\" />
-        <meta name=\"toc_rel\" content=\"toc.json\" />
```

Comment:
We still need `toc_rel` in output page meta. Does it still work?

Reply:
That's template model output, not covered by our spec test now. It's converted by integration test, check the test result